docs/cloud-edge/modules/mutual-tls.md

```diff
@@ -1,6 +1,204 @@
 # Mutual TLS
 ----------------
 
-Also known as "TLS client authentication", connections must complete a mutual TLS handshake in which the client presents a valid certificate signed by any of the root certificate authorities that you upload.
+## Overview
 
-Note: Mutual TLS applies to all routes in an HTTPS edge.
+This module performs mutual TLS (mTLS) authentication when the ngrok edge terminates
+TLS on incoming connections to your HTTP endpoint. The client must present a
+valid TLS certificate that is signed by one of the specified CAs or the
```

Comment:
Should we link specified CAs to the multiple CAs section below?

Reply:
i don't think so. it's unlikely the thing you're interested in when reading this section. using multiple CAs is relatively uncommon